proto/v1alpha1/resource.proto

```diff
@@ -51,12 +51,26 @@ message LabelTerm {
         // Label value is equal.
         EQUAL = 1;
         // Label doesn't exist.
-        NOT_EXISTS = 2;
+        NOT_EXISTS = 2 [deprecated=true];
+        // Label value is in the set.
+        IN = 3;
+        // Label value is less.
+        LT = 4;
+        // Label value is less or equal.
+        LTE = 5;
+        // Label value is less than number.
+        LT_NUMERIC = 6;
+        // Label value is less or equal numeric.
+        LTE_NUMERIC = 7;
     }
 
     string key = 1;
     Operation op = 2;
     string value = 3;
```

Comment:
I think you can add `repeated` to `string value`, and will still be backwards compatible

Reply:
gotcha